avogadro/qtplugins/importpqr/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+find_package(Qt5WebEngineWidgets REQUIRED)
```

Comment:
Please use find_package(Qt5 COMPONENTS WebEngineWidgets REQUIRED) as this only needs Qt5_DIR to be set, finding all other modules relative to that.